emuu-client/src/components/UserProfile/SubscribersList/index.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from "react";
```

Comment:
Rename this file. Index.js is the name of the file usually used for application startup.